ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -0,0 +1,391 @@
+@import '../../../styles/variables.scss';
+
+$container-width: 1200px;
+
+.main > * {
+    width: 100%;
+}
+
+section {
+    box-sizing: border-box;
+    text-align: center;
+
+    &:nth-of-type(even) {
+        background-color: #F6F7FA;
+    }
+
+    &.introduction {
+        max-width: 1440px;
```

Comment:
do we need it?

Reply:
We do. It is a requirement from our design mockup.